content/topics/dotenv-module/_index.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Introduction to Dotenv module 
+ready: true
+
+---
+
+## What is an environment variable?
+
+An [environment variable](https://en.wikipedia.org/wiki/Environment_variable) is a variable whose value is set outside the program, typically through functionality built into the operating system or [container](https://www.docker.com/resources/what-container). An environment variable is made up of a name/value pair, and any number may be created and available for reference at a point in time.
```

Comment:
Don't add container stuff here. They'll fall down a rabbit hole instead of getting the point.

The reason it works with containers is because of Operating Systems. We could link to raspberry-pi here and it would be as relevant. Stay on point. This is a focused discussion that is supposed to develop a specific skill.